### PR TITLE
Remove SSL configuration; rely on crypto-policies

### PR DESCRIPTION
**Pull request by cipherboy**
When TLSv1.3 support landed in Fedora and RHEL, crypto-policies enabled
it everywhere including in FIPS mode. However, because we bounded the
range above by TLSv1.2, we wouldn't negotiate TLSv1.3 when communicating
with CA instances. crypto-policies should be the single source of truth
for these values, and we shouldn't limit ourselves artificially.
Instead, users should change crypto-policies to the correct policy for
their needs.

`Signed-off-by: Alexander Scheel <ascheel@redhat.com>`

--- 

This is the backport of a single commit from #537 that fixes the related BZ.

### CODE REVIEW

**Comment by cipherboy** [comment]
This is in conjunction with https://github.com/dogtagpki/jss/pull/627. Without it, this PR will fail to build, as seen here. We could potentially make JSS do PHA by default, in which case, this PR will build, but IPA install will still fail.

Thoughts?

**Comment by cipherboy** [comment]
JSS PR merged. @edewata -- this is ready for review.

**Review by edewata — APPROVED** [review]
We don't really need to review backports, but here's the ACK :)
Please wait for blocker approval before merging (so we don't have to revert it if it's not approved).

**Pull request merged**
